app/i18n/fr/sub.php

```diff
@@ -127,12 +127,12 @@
 				'xpath' => 'XPath pour :',
 			),
 			'json_dotpath' => array(
-				'_' => 'JSON (Chemin)',
+				'_' => 'JSON (dot notation)',	// TODO
 				'feed_title' => array(
 					'_' => 'titre de flux',
 					'help' => 'Exemple : <code>meta.title</code> ou un texte statique : <code>"Mon flux personnalisé"</code>',
 				),
-				'help' => 'Un chemin JSON utilise le point comme séparateur objet, et des crochets pour un tableau : (ex : <code>data.items[0].title</code>)',
+				'help' => 'A JSON dot notated uses dots between objects and brackets for arrays (e.g. <code>data.items[0].title</code>)',	// TODO
```

Comment:
```suggestion
				'help' => 'La notation point pour JSON utilise le point comme séparateur objet, et des crochets pour un tableau : (ex : <code>data.items[0].title</code>)',
```

https://developer.mozilla.org/fr/docs/Learn/JavaScript/Objects/JSON